.github/workflows/e2e-test-train-api.yaml

```
@@ -0,0 +1,59 @@
+name: E2E Test with train API
+on:
+  - pull_request
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  e2e-test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        kubernetes-version: ["v1.27.11", "v1.28.7", "v1.29.2"]
```

```suggestion
        kubernetes-version: ["v1.29.2"]
```

I'm wondering if we can perform testing only for one K8s version since we already performed basic testing for every supported version.
@andreyvelich WDYT?

---

Yes, @helenxie-bit let's just use single K8s version for now. We verify other versions for the rest E2Es.

---

Got it! I will adjust accordingly.